### DISCUSSION TITLE
New template of Rooms card

### DISCUSSION THREAD

**Discussion by Ot4eVash** [Tasshack/dreame-vacuum#413] [Q&A] [answered]
The standard card template for rooms does not include an option to set the room cleaning mode (https://github.com/Tasshack/dreame-vacuum/blob/master/docs/room_entities.md). Are there any plans to update the card template? Or if anyone has already done something similar, please share!
Thank you in advance!

**Comment by Tasshack** [comment, marked as answer]
Check out the `dev` docs.
https://github.com/Tasshack/dreame-vacuum/blob/dev/docs/room_entities.md#rooms-card

**Comment by Ot4eVash** [comment]
Great thanks!!!
